tests/rt.conf

```diff
@@ -39,39 +39,39 @@ RUN | cpld_restart_c192_p8                              | - wcoss2 jet acorn gae
 RUN | cpld_bmark_p8                                     | - s4 jet cheyenne acorn    | baseline |
 RUN | cpld_restart_bmark_p8                             | - s4 jet cheyenne acorn    |          | cpld_bmark_p8
 
-COMPILE | 3 | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
+COMPILE | p8_s2sw | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
 RUN | cpld_control_noaero_p8                            |                            | baseline |
 RUN | cpld_control_nowave_noaero_p8                     |                            | baseline |
 
-COMPILE | 4 | intel | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | - wcoss2 acorn | fv3 |
+COMPILE | p8_s2swa_debug | intel | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | - wcoss2 acorn | fv3 |
 RUN | cpld_debug_p8                                     | - wcoss2 acorn             | baseline |
 
-COMPILE | 5 | intel | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | - wcoss2 acorn | fv3 |
+COMPILE | p8_noaero_debug | intel | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | - wcoss2 acorn | fv3 |
 RUN | cpld_debug_noaero_p8                              | - wcoss2 acorn             | baseline |
 
 # Waves and aerosol off for computing fluxes in mediator
-COMPILE | 6 | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_sfcocn -DCMEPS_AOFLUX=ON | | fv3 |
+COMPILE | p8_s2s_cmeps | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_sfcocn -DCMEPS_AOFLUX=ON | | fv3 |
 RUN | cpld_control_noaero_p8_agrid                      |                            | baseline |
 
-COMPILE | 7 | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
+COMPILE | p8_s2s | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
 RUN | cpld_control_c48                                  |                            | baseline |
 RUN | cpld_warmstart_c48                                |                            | baseline |
 RUN | cpld_restart_c48                                  |                            |          | cpld_warmstart_c48
 
-COMPILE | 8 | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp -DFASTER=ON | - cheyenne | fv3 |
+COMPILE | p8_faster | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp -DFASTER=ON | - cheyenne | fv3 |
```

Comment:
I'm not sure about some of your name choices. I would never associate a compile called "p8_faster" with the fully coupled s2swa compile. Or p8_pdlib with the s2sw using unstructured ww3 meshes.

If I understand, the tests directory will contain exe files that are named ``p8_faster.exe`` and ``p8_pdlib.exe``?

Reply:
No, it'll be `fv3_p8_faster_intel.exe`. I don't know what some of these executables are. If you can suggest better names, I'd appreciate it.

Reply:
You've made reasonable choices for the most part. But instead of ``p8_faster``, I'd use ``p8_s2swa_faster``.

I think instead of ``p8_s2s_cmeps``, I'd call it ``p8_s2s_aoflux``

Reply:
I've made these changes, but I'm leaving this conversation unresolved since you asked other questions.

Reply:
Not really important, but I think the first part of 'compile' name, which will be later used to generate the executable name, should be the 'APP' name, then other build options, maybe physics, or debug or 32bit etc. So in this case instead of p8_s2s_aoflux I would use s2s_p8_aoflux. It seems more logical to me. 
Also at some point we should rename all executable to start with `ufs_` instead `fv3_`, but I think we discussed this is some other issue or PR.

Reply:
I just changed it to use the app name as the first part of the compilation name. Also, I got rid of "p8" since it was everywhere, and that seems redundant.

Reply:
@DusanJovic-NOAA @DeniseWorthen are you satisfied with this solution? Trying to get these issues resolved as we finish up testing.

Reply:
Yes, I'm OK w/ the new names now.

Reply:
:+1: